### PR TITLE
C/lerna hoist

### LINKED ISSUE #363: Update to Lerna 3 and/or use hoisting?
It seems like lerna's [hoisting](https://github.com/lerna/lerna/blob/2.x/doc/hoist.md) could solve the problem of not being able to symlink hub.js packages b/c the `postinstall` step fails due to missing `devDependencies`.

Also, lerna 3 has been out for years now and the [upgrade seems fairly painless](https://samhogy.co.uk/2018/08/lerna-version-3-upgrade-guide.html). We should upgrade.

### PR DESCRIPTION
**Pull request by tomwayson**
resolves #363 

after merging this we'll be able to `npm link` Hub.js packages from the ember monorepo w/o anything like `yalc`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,7 +65,6 @@
   },
   "dependencies": {
     "@types/adlib": "^3.0.1",
-    "adlib": "^3.0.4",
```
Comment (tomwayson) [inline package.json:68]: @dbouwman I assume this was a mistake to have `adlib` installed at the root. It's not used by any of the doc tooling or anything is it?

File: package.json [hunk 2]
```diff
@@ -87,7 +86,9 @@
   },
   "scripts": {
     "build": "lerna run build",
-    "clean": "rm -rf packages/*/dist/",
+    "clean": "npm run clean:dist & npm run clean:deps",
+    "clean:dist": "rm -rf packages/*/dist/",
+    "clean:deps": "rm -rf node_modules && rm -rf packages/*/node_modules/",
```
Comment (tomwayson) [inline package.json:91]: added a new script to clean all node_modules

**Review by drewctate — APPROVED** [review]
Amazing.

**Comment by tomwayson** [comment]
Thx, but I seem to have broken the docs :(

File: package.json (hunk 1 above)
Comment (dbouwman) [reply package.json:68]: I dont think so... likely a mistake

**Comment by tomwayson** [comment]
OK, the docs are now working:

image

**Review by dbouwman — APPROVED** [review]
awesome

**Pull request merged**